### PR TITLE
Laravel 11 compatibility (with laravel-doctrine/orm laravel-11 branch)

### PR DESCRIPTION
**Pull request by 77web**
forked from #134
related https://github.com/laravel-doctrine/orm/pull/575

### CODE REVIEW

**Comment by 77web** [comment]
I changed dependant laravel-doctrine/orm branch: https://github.com/laravel-doctrine/orm/pull/575 and fixed builds!

**Comment by 77web** [comment]
`Could not authenticate against github.com` 🤔

File: composer.json [hunk 1]
```diff
@@ -1,46 +1,52 @@
...
+    "authors": [
+        {
+            "name": "Patrick Brouwers",
+            "email": "patrick@maatwebsite.nl"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/77web/laravel-doctrine-orm.git"
```
Comment (eigan) [inline composer.json]: 👀

Comment (77web) [reply composer.json]: to use laravel-doctrine/orm v11 https://github.com/laravel-doctrine/orm/pull/575

Comment (eigan) [reply composer.json]: I see 😄 I have tagged 2.1.0 in laravel-doctrine now, so you should be able to use that 🙂

Comment (77web) [reply composer.json]: https://github.com/laravel-doctrine/orm/pull/575 is merged right now, so I switched back the dependency to laravel-doctrine/orm [72afa16](https://github.com/laravel-doctrine/migrations/pull/135/commits/72afa16587694b3b3e6195711db6d1a08fd37241)

File: composer.json [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+        "migrations"
+    ],
+    "authors": [
+        {
+            "name": "Patrick Brouwers",
+            "email": "patrick@maatwebsite.nl"
+        }
+    ],
+    "require": {
+        "php": "^7.4|^8",
```
Comment (eigan) [inline composer.json]: laravel-doctrine/orm 2.0 only supports PHP 8+ :)

Comment (77web) [reply composer.json]: That's great :grin:
[930e094](https://github.com/laravel-doctrine/migrations/pull/135/commits/930e094cc171d3b5a09c6c6398caa0f480a1791f)

**Comment by eigan** [comment]
Thanks @77web 🎉

**Comment by sakowicz** [comment]
@eigan When can we expect the release with these changes?

**Comment by eigan** [comment]
@sakowicz Simply forgot, sorry. Should be out with 3.2.0 now.

**Pull request merged**